samples/nrf9160/http_update/full_modem_update/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -20,22 +15,9 @@
 	};
 };
 
-&pinctrl {
-	spi3_default_alt: spi3_default_alt {
-		group1 {
-			psels = <NRF_PSEL(SPIM_SCK, 0, 13)>,
-				<NRF_PSEL(SPIM_MOSI, 0, 11)>,
-				<NRF_PSEL(SPIM_MISO, 0, 12)>;
-		};
```

Comment:
I don't understand these pinctrl deletions, are you willing to explain? I don't question them, just want to learn